### DISCUSSION TITLE
Installing on Mac.  No instruction on what or how to use unicorn

### DISCUSSION THREAD

**Discussion by jeffmoore64** [assafelovic/gpt-researcher#113] [Q&A]
Can someone provide me with details on how to get the below command to run?

bash-3.2$ $ uvicorn main:app --reload
bash: $: command not found

**Comment by jeffmoore64** [comment]
Installed Uvicorn still get the error command not found.  Uvicorn docs don't seem to help resolve this.

**Reply by devon-ye** [reply]
## step 1： install  pyenv
``` shell
     brew install pyenv
```

## step 2: config shell env


``` shell
     echo 'eval "$(pyenv init --path)"' >> ~/.bash_profile
     echo 'eval "$(pyenv init -)"' >> ~/.bash_profile

     source ~/.bash_profile
     source ~/.zshrc
```



## step 3:check pyenv 


``` shell
   pyenv --version
```


## step 4: python  install 3.11.6  version 

``` shell
    pyenv install 3.11.6
```

## step 5: setting python  global    version  with 3.11.6

``` shell
    pyenv global 3.11.6
```
## step 6: install dependency packages

``` shell
    pip install -r requirements.txt
```
